cdap-archetypes/cdap-mapreduce-archetype/README.rst

```diff
@@ -0,0 +1,54 @@
+===========================================================
+Cask Data Application Platform (CDAP) Application Archetype
+===========================================================
+
+This directory contains a CDAP Application Archetype for the Cask Data Application Platform (CDAP)
+that includes a MapReduce program.
+
+
+Installing
+==========
+
+To install the archetype, enter in the CDAP root directory::
+
+  $ mvn clean install -f cdap-archetypes/cdap-mapreduce-archetype/pom.xml
+
+
+Creating
+========
+
+To create a project from the archetype, use this script as an example
+(substituting for ${project.version} and ${your.hadoop.version} as appropriate)::
+
+  mvn archetype:generate 					
+    -DarchetypeGroupId=co.cask.cdap 			
+    -DarchetypeArtifactId=cdap-mapreduce-archetype
+    -DarchetypeVersion=${project.version}
```

Comment:
The archetypeVersion should be the version of the cdap archetype, which is 3.3.0-SNAPSHOT in this case and become 3.3.0 when released. The version that user provides for his own project is under `-Dversion`